bokeh/core/_templates/autoload_js.js

```diff
@@ -103,9 +105,16 @@ calls it with the rendered model.
   ];
 
   function run_inline_js() {
-    for (var i = 0; i < inline_js.length; i++) {
-      inline_js[i](window.Bokeh);
-    }
+    if (window.Bokeh !== undefined || force === "1") {
+      for (var i = 0; i < inline_js.length; i++) {
+        inline_js[i](window.Bokeh);
+      }
+    } else if (Date.now() < (window._bokeh_init_load+10000)) {
+      setTimeout(run_inline_js, 500);
```

Comment:
Not quite sure how that will work. First I'd have to keep handles on all the intervals somewhere, and if I just cleared them all when window.Bokeh is defined that means they would no longer execute. Maybe I'm missing something though.